x-pack/solutions/security/packages/kbn-securitysolution-endpoint-exceptions-common/api/create_endpoint_list_item/create_endpoint_list_item.schema.yaml

```diff
@@ -0,0 +1,85 @@
+openapi: 3.0.0
+info:
+  title: Create endpoint list item API endpoint
+  version: '2023-10-31'
+paths:
+  /api/endpoint_list/items:
+    post:
+      x-labels: [serverless, ess]
+      x-codegen-enabled: true
+      operationId: CreateEndpointListItem
+      summary: Create an endpoint exception list item
+      description: Create an endpoint exception list item, and associate it with the endpoint exception list.
+      requestBody:
+        description: Exception list item's properties
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                item_id:
+                  $ref: '../../../../../../kbn-securitysolution-exceptions-common/api/model/exception_list_common.schema.yaml#/components/schemas/ExceptionListItemHumanId'
```

Comment:
The `kbn-securitysolution-exceptions-common` package got moved alongside this one so I think these refs should be `../../../kbn-securitysolution-exceptions-common`